src/core/Mixer.cpp

```diff
@@ -74,8 +74,8 @@ Mixer::Mixer( bool renderOnly ) :
 	m_framesPerPeriod( DEFAULT_BUFFER_SIZE ),
 	m_inputBufferRead( 0 ),
 	m_inputBufferWrite( 1 ),
-	m_readBuf( NULL ),
-	m_writeBuf( NULL ),
+	m_outputBufferRead( NULL ),
+	m_outputBufferWrite( NULL ),
```

Comment:
Additionally should update any `NULL` references for this to `nullptr`.
```suggestion
	m_outputBufferRead(nullptr),
	m_outputBufferWrite(nullptr),
```